sunpy/spectra/spectrogram.py

```diff
@@ -27,6 +27,8 @@
 from sunpy.spectra.spectrum import Spectrum
 from sunpy.extern.six.moves import zip, range
 
+from details_plots import log_10_product
```

Comment:
There is no module details_plots in sunpy.  Please copy the function into this PR from details_plots.